### ISSUE TITLE
BOB Button Sending Out Notifications?

### ISSUE THREAD

**Issue by emilypearl1** [dhenry-KCI/FredCo-Post-Go-Live-#413]
We dont believe that the "BOB" Button is sending out notifications? 

263132
263333
257809

Bob had an electrician tell him he didnt know he had an inspection. Bob hit the "Bob" button in the morning. The electrican realized he had an inspection  when he received his failed inspection notification.

**Comment by megisys** [comment]
Notification issues this morning.. Let me know if this still occurs.

**Comment by ABonomo21** [comment]
No log showed up for notifications so we are not sure if it works. Is there a clear way for us to know if the notification sent? @megisys

**Comment by megisys** [comment]
Notification Log will now show up in Production. = Thanks

**Comment by JuanitaMiles** [comment]
@ABonomo21 @TheBobEnsor please confirm logs are now showing up and notifications are being sent from the "send today's notifications", aka BOB button.

**Comment by ABonomo21** [comment]
![image](https://user-images.githubusercontent.com/47611580/75070815-04415300-54c2-11ea-9bf5-12d231af48f9.png)

confirmed 202341

**Comment by JuanitaMiles** [comment]
REOPENING TICKET - 

Bob Ensor has noticed that email notifications are not being sent to customers when he uses the "Send Today's Notifications" button on the Building Permits Schedule Inspections window. He has been in contact with customers who have told him they have not received the "Inspection Scheduled" notification. 

Permit 269083 is an example. It appears that pushing the "Send Today's Notifications" button does create an INSPBOOK log on the permit, but there is no NOTIFY log that indicates the notification was sent. In addition, we looked at the communications on the contact record of Ceco Inc and noticed no "inspection scheduled" type of notifications were sent to this contact since 3/30/20. The only inspection related notifications being received are inspection results. I've included screenshots showing that no inspection scheduled notification were sent on the two days the INSPBOOK log was created. (4/16 & 4/21)

![image](https://user-images.githubusercontent.com/31770504/79881422-26baf400-83bf-11ea-861b-748d962c7642.png)

![image](https://user-images.githubusercontent.com/31770504/79881602-684b9f00-83bf-11ea-9f24-b9bd7f164f59.png)

![image](https://user-images.githubusercontent.com/31770504/79881775-a8128680-83bf-11ea-9cd0-c46babaf9d1f.png)
